module/VuFind/src/VuFind/Db/Entity/ResourceTagsEntityInterface.php

```diff
@@ -40,4 +42,90 @@
  */
 interface ResourceTagsEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get resource.
+     *
+     * @return ResourceEntityInterface
+     */
+    public function getResource(): ResourceEntityInterface;
```

Comment:
The resource_id field is nullable, so the code should reflect that (`?ResourceEntityInterface` return type, etc.).